src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -106,6 +122,14 @@ def register_options(cls, register):
                 "`python_distribution`'s `entry_points` field."
             ),
         )
+        register(
+            "--unowned-dependency-behavior",
+            type=UnownedDependencyUsage,
+            default=UnownedDependencyUsage.DoNothing,
+            help=("How to handle inferred dependencies that don't have an unambiguous owner."),
+        )
+        # @TODO: Add a way for people to whitelist ok unowned dependencies?
+        # Or just use python_requirement()?
```

Comment:
Opening comment for TODO.

FWIW the following monstrosity "works" for silencing (and would represent some magical system or runtime dependency)

```
python_requirement(
    name="whatever",
    modules="modname",
    requirements=[],
```

Reply:
Yea, I think that this is worth adding an option for! There is a slightly odd asymetry in that the `module_mapping` is declared on `python_requirements` targets, while this would be declared as an option. But I don't see where else it would be reasonable to do so!

Reply:
Hmm yeah, that asymnmetry is odd. Perhaps `module_mapping` might be promoted in the future into `pants.toml` :thinking: 

In the meantime this option will certainly live there.

Reply:
I'm going to punt on this in favor of adding it later if someone requests it.